### ISSUE TITLE
Sanity check 2.4.3 Focus Order and cookie notification at bottom gets first focus

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2016]
When a cookie notification is shown at the bottom (fixed overlay over footer) and gets first focus after load, will this be a fail or can it be placed under "it does not really affect meaning or operation" so it will pass" (although not maybe the most optimal solution...?)

**Comment by jake-abma** [comment]
relates to: #2015

**Comment by fstrr** [comment]
Based on the discussion and the closed related issue, this looks like it can be closed. If it needs to be re-opened, please do.
